core/src/main/java/de/danoeh/antennapod/core/service/playback/PlaybackService.java

```diff
@@ -250,7 +250,8 @@ public void onCreate() {
         taskManager = new PlaybackServiceTaskManager(this, taskManagerCallback);
         mediaPlayer = new PlaybackServiceMediaPlayer(this, mediaPlayerCallback);
 
-        ComponentName eventReceiver = new ComponentName(getPackageName(), MediaButtonReceiver.class.getName());
+        ComponentName eventReceiver = new ComponentName(MediaButtonReceiver.class.getPackage().getName(),
+                MediaButtonReceiver.class.getName());
```

Comment:
> My computer is somewhat limited 

Only have a Core 2 Duo myself, AS's emulator is a complete mess. Genymotion works pretty well, though. (The trick is to set each devices cores to at most 2, even Lollipop is quite fast, then)